main.tf

```diff
@@ -16,6 +16,8 @@ locals {
 
   create_outposts_local_cluster    = length(var.outpost_config) > 0
   enable_cluster_encryption_config = length(var.cluster_encryption_config) > 0 && !local.create_outposts_local_cluster
+
+  tags = merge(var.tags, { terraform-aws-modules = "eks" })
```

Comment:
Should the module be forcing tags like this? From a cursory search only the `alb` and `emr` modules are currently doing this and I am curious if this is going to be a standard practice moving forward?

Reply:
It will be standard going forward to drive support for the modules

Reply:
I am not sure how adding this tag will drive support but if we are going to have a tag forced onto the resources it would be great to somehow include the module version so that consumers may also be able to benefit from tracking module and module version usage.

I also have concern around consumers with strict tagging policies and having no way to adjust/remove this to align with organizational requirements.